### PR TITLE
Added a barnyard subdomain

### PR DESCRIPTION
**Pull request by croepha**
Added a subdomain: barynyard.noisebridge.io/org/net we
delegate to brony.noisebridge.io which is currently
authortitative for those domains

### CODE REVIEW

**Comment by croepha** [comment]
See companion commit here: https://github.com/noisebridge/UnicornMigration/commit/eb3297a4a6a633f26930b76bd2e70a74b7c4aabe

**Comment by SuperQ** [comment]
Sure, but why? If you want to add more services, why not just add `CNAME` records here to brony?

**Comment by croepha** [comment]
So, I don't have a fully developed idea of what the current issues are, I'm mostly just hearing rumors, and trying help enable my fellow hackers while limiting my direct involvement... but it's clear that a pull request workflow that involves ansible isn't really working for people who are actively working on new services at Noisebridge today.  I really don't know if this will be any better, but I think it would be good to try an additional workflow...  Pure speculation here: but I think that when someone is working on trying to get something running, the last thing they want to do ramp up on Ansible or submit a pull request and wait days or weeks to get any real progress.  I think this is especially exacerbated by the fact that work on Brony/Unicorn/(or most things Noisebridge actually) is highly iterative, and I think people want quick feedback on changes.  And I think another major thing that my strategy has going for it, is that it is very approachable to add a line to file, restart the service, and be able to see the results... having direct access to the output of the DNS server is also great for debugging.  

So, in summary, I think that having a subdomain is a great sandbox for this kind of learning.  And if it doesn't solve the problem, then oh well, it doesn't cost anything to try it... Also I think that having a subdomain is a good compromise over someone going rouge and just registering yet another top level domain for Noisebridge...

**Comment by SuperQ** [comment]
"Ramping up on Ansible" really isn't an issue. All people really need to do is submit a PR with the one-line change to DNS they want. People like me can do the actual push.

> And I think another major thing that my strategy has going for it, is that it is very approachable to add a line to file, restart the service, 

IMO what you're adding is actually _harder_ than Ansible. Since you have to know how to restart the service. The Ansible deploy does this for you.

Maybe what we really need is to add a CD pipeline that takes care of restarting things.

I get the idea for a playground area, but forking our infra makes things fragile. And now you have to onboard to two different styles of managing services within NB. Which is exactly how we got into the situation of having to do a ton of work to migrate from Unicorn to Brony in the first place.

**Review by SuperQ — CHANGES_REQUESTED** [review]
Some change requests
* Update the Serial
* Put the entries in alphabetical order within their blocks.
* Drop the change from `noisebridge.org`, this is reserved for 501c3-related, not general use.

**Comment by SuperQ** [comment]
I'm fine adding this, I just think you should reconsider your plan as it makes Noisebridge infra harder to manage and maintain.

**Comment by tmelanson17** [comment]
Speaking both frankly and as a casual to server maintenance, I have no idea how Ansible works, and it seems like swatting a fly with a trebuchet given we're not providing large-scale services.
 Also, we couldn't make changes to the main repo for a couple months because "people like you" weren't around to do the push.

**Comment by SuperQ** [comment]
Well, I'm not a noob to server maintenance. I can tell you from experience it's not difficult and solves a ton of problems.

It makes it easy to juts not have to remember to do things, because you only ever have to remember one "deploy" command.

You're literally making an Ansible change in this PR. You just don't know it.

I was around, there was nothing to push.

**Comment by croepha** [comment]
Addressed comments, rebased and squashed.  Final?

**Review by SuperQ — APPROVED** [review]
LGTM

**Comment by SuperQ** [comment]
Deployed.
